lighthouse-core/test/audits/autocomplete-test.js

```diff
@@ -267,4 +367,158 @@ describe('Best Practices: autocomplete audit', () => {
     const {score} = Autocomplete.audit(artifacts);
     expect(score).toBe(1);
   });
+  it('fails when autocomplete is valid but prefix is invalid', () => {
+    const artifacts = {
+      FormElements: [
+        {
+          inputs: [
+            {
+              id: '',
+              name: 'name_cc2',
+              placeholder: '',
+              autocompleteProp: '',
+              autocompleteAttr: 'sectio-red cc-name',
+              autofillPredict: 'HTML_TYPE_CREDIT_CARD_NAME_FULL',
+              nodeLabel: 'textarea',
+              snippet: '<textarea type="text" name="name_cc2" autocomplete="sectio-red cc-name">',
+            },
+            {
+              id: '',
+              name: 'CCNo2',
+              placeholder: '',
+              autocompleteProp: '',
+              autocompleteAttr: 'shippin name',
+              autofillPredict: 'NAME_FULL',
+              nodeLabel: 'input',
+              snippet: '<input type="text" name="CCNo2" autocomplete="shippin name">',
+            },
+          ],
+          labels: [],
+        },
+      ],
+    };
+    const expectedDetails = {
+      headings: [
```

Comment:
the longer a test is, the harder it is to understand. you could completely remove all the `headings` assertions without losing any test (meaningful) coverage, and the tests would be much shorter and easier to read. mostly, you just want to test `.details.items`. but you can go further based on what the test case is for ... here, for `fails when autocomplete is valid but prefix is invalid`, you could get away checking the score.